sdk/python/jobs/finetuning/standalone/model-as-a-platform/chat/chat_completion_with_model_as_platform.ipynb

```diff
@@ -0,0 +1,565 @@
+    "* Connect to AzureML Workspace. Learn more at [set up SDK authentication](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-setup-authentication?tabs=sdk). Replace  `<WORKSPACE_NAME>`, `<RESOURCE_GROUP>` and `<SUBSCRIPTION_ID>` below.\n",
+    "* Connect to `azureml` system registry\n",
+    "* Set an optional experiment name"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Install dependencies by running below cell. This is not an optional step if running in a new environment.**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%pip install https://finetuningsdkteststorage.blob.core.windows.net/finetuning-private-sdk/azure_ai_ml-1.23.0-py3-none-any.whl\n",
+    "%pip install azure-identity\n",
+    "\n",
+    "%pip install mlflow\n",
+    "%pip install azureml-mlflow"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml import MLClient\n",
+    "from azure.identity import (\n",
+    "    DefaultAzureCredential,\n",
+    "    InteractiveBrowserCredential,\n",
+    ")\n",
+    "from azure.ai.ml.constants._common import AssetTypes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create AzureML Workspace connections"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml import MLClient\n",
+    "from azure.identity import (\n",
+    "    DefaultAzureCredential,\n",
+    "    InteractiveBrowserCredential,\n",
+    ")\n",
+    "\n",
+    "try:\n",
+    "    credential = DefaultAzureCredential()\n",
+    "    credential.get_token(\"https://management.azure.com/.default\")\n",
+    "except Exception as ex:\n",
+    "    credential = InteractiveBrowserCredential()\n",
+    "\n",
+    "try:\n",
+    "    workspace_ml_client = MLClient.from_config(credential=credential)\n",
+    "except:\n",
+    "    workspace_ml_client = MLClient(\n",
+    "        credential,\n",
+    "        subscription_id=\"<SUBSCRIPTION_ID>\",\n",
+    "        resource_group_name=\"<RESOURCE_GROUP>\",\n",
+    "        workspace_name=\"<WORKSPACE_NAME>\",\n",
+    "    )\n",
+    "\n",
+    "# the models, fine tuning pipelines and environments are available in various AzureML system registries,\n",
+    "# Example: Phi family of models are in \"azureml\"\n",
+    "registry_ml_client = MLClient(credential, registry_name=\"azureml\")\n",
+    "\n",
+    "# Get AzureML workspace object.\n",
+    "workspace = workspace_ml_client._workspaces.get(workspace_ml_client.workspace_name)\n",
+    "workspace.id"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2. Pick a foundation model to fine tune\n",
```

Comment:
There is no reference to 'foundational' in UI. What's the reason to use the term here ?